assets/css/style-freenet-3.css

```diff
@@ -91,6 +91,14 @@ h4 {
 }
 
 /* =============================================================
+OVERRIDE GLOBAL COLOR FOR MODAL DIALOG
+============================================================ */
+
+.modal-content {
```

Comment:
To ensure the CSS isn't cached without this rule, could you rename it to `style-freenet-4.css` and replace its references?